src/components/surveys/survey-design/question-edit/rhs-subcontrols/Time.tsx

```diff
@@ -129,7 +129,11 @@ const Time: React.FunctionComponent<{
     if (val) {
       setRange(undefined)
       setError('')
-      //onUpdateFormat(undefined)
+      onUpdateFormat({
+        ...step.inputItem.formatOptions,
+        maximumValue: undefined,
+        minimumValue: undefined,
+      })
```

Comment:
After checking "No min or max validation", update the step to reflect that there is no `minimumValue` or `maximumValue` set.

<img width="455" alt="image" src="https://github.com/BridgeDigitalHealth/open-bridge-web/assets/26949006/d7a387a8-4631-43d0-bde2-26c674dd2fad">